src/PresentationalComponents/ZeroState/ZeroState.js

```diff
@@ -96,49 +92,8 @@ const ZeroState = () => {
     }, []);
 
     return <div className='insd-c-marketing-page'>
-        <ZeroStateBanner/>
-        <PageSection isWidthLimited>
-            <Grid lg={ 6 } hasGutter>
-                <GridItem>
-                    <Card style={ { height: '100%' } }>
-                        <CardHeader>
-                            <TextContent>
-                                <Title headingLevel='h2'>{intl.formatMessage(messages.zslblTitle)}</Title>
-                                <p>{intl.formatMessage(messages.zslblBody)}</p>
-                            </TextContent>
-                        </CardHeader>
-                        <CardBody>
-                            <IconList>
-                                <IconListItem>{intl.formatMessage(messages.zslblb1)}</IconListItem>
-                                <IconListItem>{intl.formatMessage(messages.zslblb2)}</IconListItem>
-                                <IconListItem>{intl.formatMessage(messages.zslblb3)}</IconListItem>
-                                <IconListItem>{intl.formatMessage(messages.zslblb4)}</IconListItem>
-                                <IconListItem>{intl.formatMessage(messages.zslblb5)}</IconListItem>
-                            </IconList>
-                        </CardBody>
-                    </Card>
-                </GridItem>
-                <GridItem>
-                    <Card style={ { height: '100%' } }>
-                        <CardHeader>
-                            <TextContent>
-                                <Title headingLevel='h2'>{intl.formatMessage(messages.zsrblTitle)}</Title>
-                                <p>{intl.formatMessage(messages.zsrblBody)}</p>
-                            </TextContent>
-                        </CardHeader>
-                        <CardBody>
-                            <IconList>
-                                <IconListItem>{intl.formatMessage(messages.zsrblb1)}</IconListItem>
-                                <IconListItem>{intl.formatMessage(messages.zsrblb2)}</IconListItem>
-                                <IconListItem>{intl.formatMessage(messages.zsrblb3)}</IconListItem>
-                                <IconListItem>{intl.formatMessage(messages.zsrblb4)}</IconListItem>
-                                <IconListItem>{intl.formatMessage(messages.zsrblb5)}</IconListItem>
-                            </IconList>
-                        </CardBody>
-                    </Card>
-                </GridItem>
-            </Grid>
-        </PageSection>
+        <ZeroStateBanner />
```

Comment:
Zero state on dashboard is going to stay the same for the time being until UX/PMs agree on copy. Its part of P3. For now we gotta focus on the federated modules to the individual apps